Files/Interacts/Interaction.cs

```diff
@@ -1095,6 +1108,12 @@ public async Task PasteItems(DataPackageView packageView, string destinationPath
                         // File was moved/deleted in the meantime
                         continue;
                     }
+                    catch (Exception ex)
```

Comment:
I think there is better way to detect an exception.

Reply:
The thrown exception in case of resource in use is of type "System.Exception".